pkg/disttask/framework/handle/handle.go

```diff
@@ -267,6 +271,21 @@ func GetTargetScope() string {
 	return vardef.ServiceScope.Load()
 }
 
+// GetCloudStorageURI returns the cloud storage URI with cluster ID appended to the path.
+func GetCloudStorageURI(ctx context.Context, store kv.Storage) string {
+	cloudURI := vardef.CloudStorageURI.Load()
+	if s, ok := store.(kv.StorageWithPD); ok {
+		// When setting the cloudURI value by SQL, we already checked the effectiveness, so we don't need to check it again here.
+		u, _ := litstorage.ParseRawURL(cloudURI)
+		if len(u.Path) != 0 {
+			u.Path = path.Join(u.Path, strconv.FormatUint(s.GetPDClient().GetClusterID(ctx), 10))
+			return u.String()
+		}
+	}
+	logutil.BgLogger().Error("Can't get cluster id from store, use default cloud storage uri")
```

Comment:
What scenario we can not get cluster id?

Reply:
In actual scenarios, the tikv is always comes with pd, so we will not meet this scenario. I wrote this to serve as a reminder and protection of the using mocked store.